examples/grpc-bridge/README.md

```diff
@@ -1,2 +1,124 @@
-To learn about this sandbox and for instructions on how to run it please head over
-to the [envoy docs](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/grpc_bridge)
+# gRPC HTTP/1.1 to HTTP/2 bridge
```

Comment:
Drive by: Please move this back to the RST file. It's substantially better for link checking, consistency, etc. Thank you!

Reply:
@mattklein123 Yeah definitely! Let me put it back!

Reply:
@snowp @marcellodesales as I asked above, can you please move all of this text to the Sphinx/RST docs? We want as little text as possible in markdown as it is much more difficult to keep links fresh, etc., and it's also more consistent. @marcellodesales a follow up PR is fine. Thank you!